client/src/app/pages/reports/components/adoption-candidate-table/adoption-candidate-table.tsx

```diff
@@ -1,252 +1,141 @@
-        //         <Label>{t("terms.notReviewed")}</Label>
-        //       )}
-        //     </>
-        //   ),
-        // },
-      ],
-    });
+    variant: "compact",
   });
 
-  // const selectRow = (
-  //   event: React.FormEvent<HTMLInputElement>,
-  //   isSelected: boolean,
-  //   rowIndex: number,
-  //   rowData: IRowData,
-  //   extraData: IExtraData
-  // ) => {
-  //   const row = getRow(rowData);
-  //   toggleApplicationSelected(row.application);
-  // };
-  const handleOnClearAllFilters = () => {
-    setFilterValues({});
-  };
+  const {
+    currentPageItems,
+    numRenderedColumns,
+    selectionState,
+    propHelpers: { tableProps, getThProps, getTrProps, getTdProps },
+  } = tableControls;
 
   return (
-    <AppTableWithControls
-      paginationProps={paginationProps}
-      paginationIdPrefix="adoption-candidate"
-      variant={TableVariant.compact}
-      count={assessments.length}
-      sortBy={sortBy}
-      onSort={onSort}
-      cells={columns}
-      rows={rows}
-      // onSelect={selectRow}
-      // canSelectAll={false}
-      isLoading={false}
-      filtersApplied={false}
-      toolbarClearAllFilters={handleOnClearAllFilters}
-      toolbarToggle={
-        <FilterToolbar
-          filterCategories={filterCategories}
-          filterValues={filterValues}
-          setFilterValues={setFilterValues}
-        />
-      }
-      // toolbarBulkSelector={
-      //   <ToolbarBulkSelector
-      //     onSelectAll={selectAllApplication}
-      //     areAllSelected={areAllApplicationsSelected}
-      //     selectedRows={selectedApplications}
-      //     paginationProps={paginationProps}
-      //     currentPageItems={currentPageItems.map((item) => item.application)}
-      //     onSelectMultiple={selectMultipleApplications}
-      //   />
-      // }
-    />
+    <Table
+      {...tableProps}
+      aria-label={`Adoption Candidate Table`}
+      style={{ background: "none" }}
+    >
+      <Thead>
+        <Tr>
+          <TableHeaderContentWithControls {...tableControls}>
+            <Th {...getThProps({ columnKey: "applicationName" })}>
+              Application Name
+            </Th>
+            <Th {...getThProps({ columnKey: "criticality" })}>Criticality</Th>
+            <Th {...getThProps({ columnKey: "priority" })}>Priority</Th>
+            <Th {...getThProps({ columnKey: "confidence" })}>Confidence</Th>
+            <Th {...getThProps({ columnKey: "effort" })}>Effort</Th>
+            <Th {...getThProps({ columnKey: "risk" })}>Risk</Th>
+          </TableHeaderContentWithControls>
+        </Tr>
+      </Thead>
+      <ConditionalTableBody
+        isNoData={allApplications?.length === 0}
+        numRenderedColumns={numRenderedColumns}
+        noDataEmptyState={
+          <div>
+            <NoDataEmptyState title="No data available." />
+          </div>
+        }
+      >
+        <Tbody>
+          {currentPageItems?.map((item, rowIndex) => {
+            return (
```

Comment:
Nit-pick -- skip the open `,rowIndex) => { return (` and just go with `, rowIndex) => (`